### DISCUSSION TITLE
How to configure font size globally

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#4912] [Help]
Hi All,

How I can set the font size globally,.
Like more smaller or bigger the font size.

Thanks
Jigu

**Comment by danharrin** [comment]
CSS in your theme file?

**Comment by jigujigu** [comment]
Hi. danharrin,
do you mean in resources\css\app.css or resources\css\filament.css ?
I can't find the configuration for font size.
Please more advice.
I am still using filament v2

Thank You
